br/pkg/glue/console_glue.go

```diff
@@ -129,8 +129,10 @@ func (t *Table) maxKeyLen() int {
 
 // Print prints the table.
 // The format would be like:
-//    Key1: <Value>
-//   Other: <Value>
+//
```

Comment:
Is this introduced by the new rule of lint tools?

Reply:
Yes, It is changed by ```gofmt``` of go1.19

Reply:
Got, I saw the updates for go1.19: https://go.dev/doc/comment